quantum/quantum_keycodes.h

```diff
@@ -454,6 +454,42 @@ enum quantum_keycodes {
     TERM_OFF,
 #endif
 
+#ifdef JOYSTICK_ENABLE
+    JS_BUTTON0 = 0x5E00,
+    JS_BUTTON1,
+    JS_BUTTON2,
+    JS_BUTTON3,
+    JS_BUTTON4,
+    JS_BUTTON5,
+    JS_BUTTON6,
+    JS_BUTTON7,
+    JS_BUTTON8,
+    JS_BUTTON9,
+    JS_BUTTON10,
+    JS_BUTTON11,
+    JS_BUTTON12,
+    JS_BUTTON13,
+    JS_BUTTON14,
+    JS_BUTTON15,
+    JS_BUTTON16,
+    JS_BUTTON17,
+    JS_BUTTON18,
+    JS_BUTTON19,
+    JS_BUTTON20,
+    JS_BUTTON21,
+    JS_BUTTON22,
+    JS_BUTTON23,
+    JS_BUTTON24,
+    JS_BUTTON25,
+    JS_BUTTON26,
+    JS_BUTTON27,
+    JS_BUTTON28,
+    JS_BUTTON29,
+    JS_BUTTON30,
+    JS_BUTTON31,
+    JS_BUTTON_MAX = JS_BUTTON31,
```

Comment:
Maybe add a 
```suggestion
JS_BUTTON_MIN = JS_BUTTON0
```

Reply:
@mechmerlin it looks like your suggestion replaces that line - I think you can make one that adds a new line as well, right?